core/model/src/payment.rs

```diff
@@ -239,6 +239,7 @@ pub mod local {
         pub driver: String,
         pub network: Option<String>,
         pub token: Option<String>,
+        pub since: i64,
```

Comment:
I'd rename this property to `age`, `last` or something similar. `since` suggests a point in time (timestamp)